analyses/interaction-plots/scripts/cooccur_functions.R

```diff
@@ -81,10 +81,14 @@ filter_mutations <- function(maf_df,
 #' @return A data frame summarizing the co-occurence of pairs of genes in the
 #'   gene list with columns `gene1`; `gene2`; counts of each mutations in
 #'   each category of sharing (`mut11`: both mutated; `mut10`: mutated in
-#'   the first but not second gene, etc.); `odds_ratio` for co-occurence,
-#'   `cooccur_sign`: 1 if co-occurence greater than by chance, -1 if less
-#'   frequent than expected; `p` the fisher's exact test p value; and a
-#'   `cooccur_score` calucated as `cooccur_sign * -log10(p)`.
+#'   the first but not second gene, etc.) and summary (`n_mutated_gene1` and 
+#'   `n_mutated_gene2`: number of samples with gene1 or gene2 mutated, ) `n_mutated_gene1`); 
+#'   `odds_ratio` for co-occurence, `cooccur_sign`: 1 if co-occurence greater than by chance, 
+#'   -1 if less frequent than expected; `p` the fisher's exact test p value; and a
+#'   `cooccur_score` calculated as `cooccur_sign * -log10(p)`. Overall percent 
+#'   of samples mutated denoted as `perc_mutated_gene1` and `perc_mutated_gene2`. 
+#'   Percent of gene1-mutated samples in which gene2 mutations are co-occuring or mutually exclusive 
+#'   denoted as `perc_cooccur_or_mutexcl`.
```

Comment:
I think we should reformat this section for easier reading/updating. The current list is not in the order of the output table, so we should also try to make that the case. I have a suggested format below, which also incorporates my suggestion to split cooccurance and mutual exclusivity into separate columns.

```
#' @return A data frame summarizing the co-occurence of pairs of genes in the
#'   gene list with columns:
#'  `gene1`; `gene2` (gene names);
#'  `mut11`; `mut10`; `mut01` (counts of mutations in each category of sharing:
#'     `mut11`: both mutated; `mut10`: mutated in the first but not second gene, etc.);
#'  `odds_ratio` (odds ratio for for co-occurence);
#'  `cooccur_sign` (1 if co-occurence greater than by chance, -1 if less frequent than expected)
#'  `p` (the fisher's exact test p value);
#'  `q` (Benjamini-Hochberg adjusted p value);
#'  `cooccur_score` (calculated as `cooccur_sign * -log10(p)`);
#'  `n_mutated_gene1`; `n_mutated_gene2` (count of samples with gene1 or gene2 mutations); 
#'  `perc_mutated_gene1`; `perc_mutated_gene2` (percent of samples with gene 1 or gene 2 mutations);
#'  `perc_cooccur`;
#'  `perc_mutexl`;
```